### PR TITLE
FOCUS #11: Provider spec changes and supporting documents

### PR DESCRIPTION
**Pull request by ijurica**
Provider spec changes and supporting documents

### CODE REVIEW

File: specification/dimensions/provider.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Provider
+
+A Provider is an entity that made the infrastructure and/or services available for purchase. It is commonly used for cost analysis and reporting scenarios.
```
Comment (flanakin) [inline specification/dimensions/provider.md:3]: nit: Is "cost analysis and reporting scenarios" helpful? We could arguably say that about every column. I feel like we could leave that sentence out and not lose anything. Is it needed or should we give more specific scenario examples?

**Review by flanakin — APPROVED** [review]

File: specification/dimensions/provider.md (hunk 1 above)
Comment (ijurica) [reply specification/dimensions/provider.md:3]: I agree.
in this chapter, in this iteration I just followed the concept used for Service Name, Resource ID... (only left out the allocation)

File: specification/dimensions/provider.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Provider
+
+## Display name
+
+Provider
+
+## Description
+
+The entity that made the infrastructure and/or services available for purchase.
```
Comment (udam-f2) [inline specification/dimensions/provider.md:19]: Nit: Should be resources and/or services here and elsewhere (you may have already said you're making that change)

Comment (ijurica) [reply specification/dimensions/provider.md]: replaced in all 3 branches (infrastructure and/or services -> resources and/or services)

File: specification/dimensions/dimensions.mdpp [hunk 3]
```diff
@@ -7,5 +7,6 @@ dimensions to categorize, filter, and reveal details in your data when grouped w
 !INCLUDE "resourceid.md",1
 !INCLUDE "resourcename.md",1
 !INCLUDE "servicename.md",1
```
[the remark this answers is not in the document]
Comment (ijurica) [reply specification/dimensions/dimensions.mdpp]: in the current version positioned above servicename (applied to all 3 dimensions/branches)

File: specification/dimensions/provider.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Provider
+
+A Provider is an entity that made the infrastructure and/or services available for purchase. It is commonly used for cost analysis and reporting scenarios.
+
+The Provider column MUST be present in the billing data. This column MUST be of type String and MUST NOT contain null values.
+
+See [Appendix: Origination of the cost data](../appendix/origination_of_cost_data.md) section for context (Provider/Publisher/Invoicing Entity) in various purchasing scenarios.
```
[the remark this answers is not in the document]
Comment (ijurica) [reply specification/dimensions/provider.md]: Checked the appendix-section branch and changed to "... See [Appendix: Origination of cost data](**#origination-of-cost-data**) ..." (applied to all 3 dimensions/branches). If needed, I can easily change it to (**#originationofcostdata**)

[the remark this answers is not in the document]
Comment (ijurica) [reply specification/dimensions/provider.md]: change applied in all 3 cases (Provider, Publisher, Invoice Issuer)

**Comment by ijurica** [comment]
Short overview of the currently applied changes and leftovers...

**APPLIED CHANGES:**

- Due to dimension renaming (Invoicing Entity -> Invoice Issuer), a mention of the Invoicing Entity in the supporting_content MD file was replaced.
- Current ordering in the dimensions.mdpp file - invoiceissuer.md placed above servicename.md 
- Two mentions of the term "infrastructure and/or services" were replaced with the term "resources and/or services".
- In the spec MD file, link to the corresponding appendix section uses anchor (#origination-of-cost-data). Can be easily changed to (#originationofcostdata).
- The proposed change, related to the wording of the reference to the appendix section in the supporting_content MD file (see conversation in #11 Provider PR), has been implemented.

**LEFTOVERS:**

- Column ID vs. Display Name within normative content - see unresolved conversation in #11 Provider PR
- Note regarding dimension's common usage scenarios - see unresolved conversation in #11 Provider PR

File: specification/dimensions/provider.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (udam-f2) [reply specification/dimensions/provider.md]: It's definitely #originationofcostdata. I tried with dashes and saw it doesn't work. Also checked the HTML and saw the a tag and it was #originationofcostdata. 

You can essentially put that link in some doc, run the make command and look at the output and see the result.

[the remark this answers is not in the document]
Comment (ijurica) [reply specification/dimensions/provider.md]: changed to (#originationofcostdata)

**Review by udam-f2 — APPROVED** [review]
LGTM

**Pull request merged**